<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlaceRecognition</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonOpenSeq">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image Sequence Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMakeDB">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectQuery">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder Name :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoadDB">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Database</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DB Name : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>102</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ImageName :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSeqPath">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>481</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelDBPath">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>421</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelQuName">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>102</y>
      <width>711</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="boxQuery">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>501</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>QueryImage</string>
    </property>
    <widget class="QLabel" name="labelQueryImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>471</width>
       <height>371</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="QueryIdx">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string>0</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="boxDB">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>501</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>DBImage</string>
    </property>
    <widget class="QLabel" name="labelDBImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>471</width>
       <height>371</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ReturnIdx">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string>0</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonQueryImage">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Query Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSaveDB">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Database</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
